ribbon-transport/src/main/java/com/netflix/ribbon/transport/netty/http/LoadBalancingHttpClient.java

```diff
@@ -511,6 +518,23 @@ ServerStats getServerStats(Server server) {
                     @Override
                     public SSLEngine createSSLEngine(ByteBufAllocator allocator) {
                         SSLEngine myEngine = super.createSSLEngine(allocator);
+
+                        URL url = null;
+                        if(null!=getRxClients()) {
+                          try {
+                            Server server = getRxClients().entrySet().iterator().next().getKey();
```

Comment:
Some dough that is correct to get only the first one

Reply:
gather all hosts